src/auto-evo/simulation/food_source/HeterotrophicFoodSource.cs

```diff
@@ -68,30 +68,12 @@ public HeterotrophicFoodSource(Patch patch, MicrobeSpecies prey, SimulationCache
                 engulfScore = catchScore * Constants.AUTO_EVO_ENGULF_PREDATION_SCORE;
             }
 
-            var pilusScore = 0.0f;
-            var oxytoxyScore = 0.0f;
-            var mucilageScore = 0.0f;
-            foreach (var organelle in microbeSpecies.Organelles)
-            {
-                if (organelle.Definition.HasPilusComponent)
-                {
-                    pilusScore += Constants.AUTO_EVO_PILUS_PREDATION_SCORE;
-                    continue;
-                }
-
-                foreach (var process in organelle.Definition.RunnableProcesses)
-                {
-                    if (process.Process.Outputs.TryGetValue(oxytoxy, out var oxytoxyAmount))
-                    {
-                        oxytoxyScore += oxytoxyAmount * Constants.AUTO_EVO_TOXIN_PREDATION_SCORE;
-                    }
-
-                    if (process.Process.Outputs.TryGetValue(mucilage, out var mucilageAmount))
-                    {
-                        mucilageScore += mucilageAmount * Constants.AUTO_EVO_MUCILAGE_PREDATION_SCORE;
-                    }
-                }
-            }
+            // The array was used to have a time & memory efficient structure,
+            // but this could be turned into a class for more readable and less error-prone code.
```

Comment:
What about a tuple with named elements? Those are used a lot in Thrive to quickly package a couple of variables into one unit.